### DISCUSSION TITLE
Auto install without initial setup menu

### DISCUSSION THREAD

**Discussion by jpatriarca** [fabianishere/udm-iptv#225] [Ideas]
Hi @fabianishere,

Now with UDM-Pro firmware change and the need to reinstall udm-iptv anytime a new firmware is installed in the device (like UDM-Pro SE), it would be possible to have a installs script on your repository that would be fully automatic and expect the settings file already in place?

This would be beneficial, for example, to benefit from bott script installed to be run on boot and automatically install without any intervention. Also it would be great, if the script would verify if udm-iptv is installed and ignore next steps.

Would be something that you're willing to create?

Thank you for your amazing work here!

**Comment by fabianishere** [comment]
I think you can already do this, if you want to automate it with a boot script. Have a look at `debconf-set-selections` to programmatically set the options for `udm-iptv`.

**Comment by jpatriarca** [comment]
Thank you @fabianishere. Your tip have helped me a lot to understand how to use that tool, capture the manual configurations and apply it for automation. Will test this approach.
